### ISSUE TITLE
[loading-image-default-eager] background-images and generated content?

### ISSUE THREAD

**Issue by Malvoz** [w3c/webappsec-feature-policy#311]
Before opening https://github.com/w3c/csswg-drafts/issues/3659, I had assumed `loading-image-default-eager` applies to CSS background-images and generated content. But I don't think that's mentioned anywhere. @ehsan-karamad have you given any thought to this?

**Comment by ehsan-karamad** [comment]
I would have to pass that question to @bengreenstein since IMO this is more of a `loading` attribute question. The policy, when disabled, should (almost always) treat unset or `auto` attribute values as `lazy`. It is almost always to open room for some user agent optimizations for certain images (e.g., small images). I am not quite sure how the `loading` attribute deals with background images.
cc @clelland.

**Comment by clelland** [comment]
I don't think that the `loading` attribute applies to CSS at all; as specced, it's only defined on `img` (and `iframe`) elements, and only affects their replaced content, not the background. If that spec were to change, (maybe by defining a parallel `background-loading` CSS property) then I think `loading-image-default-eager` should cover that as well, but that would have to be defined first.
